classes/ProductAssembler.php

```diff
@@ -81,7 +83,10 @@ private function addMissingProductFields(array $rawProduct)
                     ON pl.id_product = p.id_product
                     AND pl.id_shop = $idShop
                     AND pl.id_lang = $idLang
-                    AND p.id_product = $idProduct";
+                    AND p.id_product = $idProduct
+                LEFT JOIN {$prefix}stock_available stock
```

Comment:
In PrestaShop, table aliases are named with the first letter of each word of the table name. So, it should be `sa` for `stock_available`